backend/src/jwt/encode.spec.js

```diff
@@ -0,0 +1,62 @@
+import encode from './encode'
+import jwt from 'jsonwebtoken'
+import CONFIG from './../config'
+
+describe('encode', () => {
+  let payload
+  beforeEach(() => {
+    payload = {
+      name: 'Some body',
+      slug: 'some-body',
+      id: 'some-id',
+    }
+  })
+
+  it('encodes a valided JWT bearer token', () => {
```

Comment:
```suggestion
  it('encodes a valid JWT bearer token', () => {
```
normally, I would suggest this change, but with the build server so difficult to get passing, I think we can make this change some other time